### ISSUE TITLE
bgmm::semisupervised failing on value of parameter B which shouldn't be needed

### ISSUE THREAD

**Issue by andrewcstewart** [pbiecek/bgmm#3]
I think something is broken in semisupervised.  It's trying to use ncol(B) but B shouldn't be a parameter of that method.

```
modelSemiSupervised = semisupervised(X=genotypes$X, 
            knowns=genotypes$knowns, class = genotypes$labels)
Error in if (pca.dim.reduction < ncol(B)) { : argument is of length zero
```

**Comment by andrewcstewart** [comment]
(using the latest bgmm from devtools::install_github() btw)

**Comment by pbiecek** [comment]
Right, thanks.
Fixed with 4bb768eca56cd05678c80354f42e23133cf631ba.

**Comment by andrewcstewart** [comment]
Fix confirmed.  Thanks!
